lib/services/storageManagement/lib/storageManagementClient.js

```diff
@@ -2004,5 +2004,134 @@ var StorageAccountOperations = ( /** @lends StorageAccountOperations */ function
     });
   };
   
+    /**
+   * The Validate Network Operation validates your storage account for IaaS
+   * Classic to ARM migration.
+   * 
+   * @param {string} storageAccountName  Name of storage account to be migrated.
+   * 
+   * @param {function} callback
+   * 
+   * @returns {Stream} The response stream.
+   */
+  StorageAccountOperations.prototype.validateMigration = function (storageAccountName, callback) {
+        if (callback === null || callback === undefined) {
```

Comment:
Fixed. Auto-generated the code now. I was not aware of code auto-generator. Thanks!